etc/benchmarks/main.mjs

```diff
@@ -158,15 +158,41 @@ await runner({
 });
 
 await runner({
-  skip: true,
-  name: 'deserialize a large batch of documents each with an array of many Int64s',
+  skip: false,
+  name: 'deserialize a large batch of documents with multiple types',
```

Comment:
Instead of replacing the test that was here can we start a new one with targeting this muli-type object?

Reply:
Yep will clean everything if the perf changes are validated